File: python/shape_calculation.py
class Shape:
    length = 0
    breath = 0
    width = 0

    def __init__(self, l:int, b:int, w:int):
        self.breath = b
        self.length = l
        self.width = w

    def volume(self) -> str:
        volume = self.breath * self.length * self.width

        return f"The volume is {volume}"
    
    def area(self) -> str:
        area = self.length * self.breath

        return f"The area is {area}"
    
    def perimeter(self) -> str:
        perimeter = 2*(self.length + self.breath)

        return f"The perimeter is {perimeter}"
    

def main():
    square = Shape(4,4,4)
    area = square.area()
    perimeter = square.perimeter()

    print(f"The area and perimeter of the square are {area, perimeter}")


if __name__ == "__main__":
    main()

File: javascript/shape_calculation.js
class Shape {
    #length
    #breadth
    #width

    constructor(l, b, w){
        this.#breadth = b;
        this.#length = l;
        this.#width = w
    }

    area(){
        if(this.#length === 0 || this.#breadth === 0){
            return
        }

        console.log(`The area is: ${this.#length * this.#breadth}`)
    }

    volume(){
       if(this.#length === 0 || this.#breadth === 0 || this.#width === 0){
            return
        }

        console.log(`The volume is: ${this.#length * this.#breadth  * this.#width}`) 
    }

    perimeter(){
        if(this.#length === 0 || this.#breadth === 0){
            return
        }

        console.log(`The perimeter is: ${2 * (this.#length + this.#breadth)}`)
    }
}


function main(){
    const shape = new Shape(10, 10, 6)

    shape.area()
    shape.perimeter()
    shape.volume()
}

main()